src/__experimental__/components/label/label.component.js

```diff
@@ -14,6 +14,12 @@ const shouldDisplayValidationIcon = ({
   return disabled ? false : typeof validation === 'string';
 };
 
+const shouldDisplayInlineTooltip = ({
+  error, warning, info, inline
+}) => {
+  return (error || warning || info) && inline === true;
```

Comment:
We're returning a boolean here to then convert it to `top`/`right` below. We can simplify this by returning the tooltip position here.

```js
const tooltipPosition = ({
  error, warning, info, inline
}) => {
  return (error || warning || info) && inline === true ? 'top' :'right';
};

```

Reply:
Simplified this so the function returns the tooltip position.